tsconfig.json

```
@@ -18,7 +18,7 @@
     "esModuleInterop": true,
     "removeComments": false,
     "jsx": "preserve",
-    "lib": ["esnext", "dom"],
+    "lib": ["es2015", "dom"],
```

```suggestion
    "lib": ["es2016", "dom"],
```

Should be `es2016` to align with `target`.

---

I set `target` to `es2015` instead.

---

Vue's codebase makes heavy use of `Array.prototype.includes`, which is an ES2016 feature, so ES2016 is more accurate.